sync/cli/awsemr.py

```diff
@@ -102,6 +102,23 @@ def create_project_prediction(project: Dict[str, str], run_id: str = None, regio
         click.echo(f"Failed to create prediction. {prediction_response.error}", err=True)
 
 
+@aws_emr.command
+@click.option(
+    "--project", callback=validate_project, help="The project ID for which to submit data."
+)
```

Comment:
This is required isn't it?

Reply:
Ah, good catch. I made it an `argument` for Databricks